src/lib/mina_generators/zkapp_command_generators.ml

```diff
@@ -1127,7 +1127,7 @@ let gen_zkapp_command_from ?global_slot ?memo ?(no_account_precondition = false)
     ?(no_token_accounts = false) ?(limited = false)
     ?(generate_new_accounts = true) ?failure
     ?(max_account_updates = max_account_updates)
-    ?(max_token_updates = max_token_updates)
+    ?(max_token_updates = max_token_updates) ?(map_account_update = ident)
```

Comment:
Nit:

New line in between ?

Reply:
I'm quite sure this is managed by the code formatter